### PR TITLE
Rwahs 267/photographs   description to go into summary field

### PR DESCRIPTION
**Pull request by kehh**
Includes #18 and #19, so those should be merged first.
- Create `Summary` field for photographs based on `Description`
- Update import mapping pushing data from new `Summary` field into the `Summary` attribute

### CODE REVIEW

**Comment by kehh** [comment]
@leftclickben I've assigned to you. Once the other PR's are merged, please assign this back to me and I'll add comments if necessary.

**Comment by kehh** [comment]
I just discovered that I did not create a new branch, so all my commits after   bf4702d have been going on to the wrong branch :(

**Comment by kehh** [comment]
OK thanks to `git reset` it's possible to rewind those changes.

**Comment by kehh** [comment]
but still, let me know when you've merged #19 and I'll then update this PR with relevant comments.

**Comment by leftclickben** [comment]
#19 is merged, and there is also a conflict on this one now.

File: importScripts/sampleRecords/itemData.destination.json [hunk 1]
```diff
@@ -184,11 +191,11 @@
...
+      "ca_objects.Summary":"Rottnest settlement: Thomsons Bay 1900-1908. From right to left - The Pilot boats crew quarters-dining room, kitchen, etc. The opening in front was the boat shed. The Pilot took ships to Fremantle and he was later ferried back to the island. The second building was originally the Pilot's quarters, then the Signalmans and now Des Sullivan's house. The third building was semi-detached houses, the left one occupied by Chief Warder J.J. O'Donoghue on his retirement. In 1917 or 1918 O'Donoghue was transferred to the Board of Control. Another occupant was the school teacher. The fourth building was the Superintendents house. On the extreme left is the Governor's residence or Government House as the locals called it. The sea wall was built by Superintendent Vincent. <br \/>\nAccession No. 43.120",
```
Comment (kehh) [inline importScripts/sampleRecords/itemData.destination.json:198]: So how come this one doesn't pick up the 'Summary' delta like all the others?

**Comment by kehh** [comment]
I think it's pretty much self explanatory.

**Pull request merged**